Java/core/src/main/java/com/amazon/randomcutforest/tree/AbstractCompactRandomCutTree.java

```diff
@@ -63,12 +65,15 @@
     protected IBoxCache<Point> boxCache;
     protected Point[] pointSum;
     protected HashMap<Long, Integer>[] sequenceIndexes;
+    protected boolean float32Precision;
```

Comment:
Let's move the Precision field here and out of the concrete classes.

Reply:
Done